cha/design/sync.tex

```diff
@@ -99,14 +98,38 @@ \section{Conclusion}\label{sec:sync_conclusion}
-        Light cues & No external hardware requirements. & Require cameras to face each other, \newline hardware limitations proces further complexity, \newline requires manual work. \\
-        \ac{NTP} & High-precition clocks, \newline mature protocol. & Requires internet. \\
+        Audio   & No external hardware requirements. & Very complicated with large number of devices, \newline large synchronization time, \newline audio latency is different from device to device. \\
+        Light cues & No external hardware requirements. & Require cameras to face each other, \newline hardware limitations process further complexity, \newline requires manual work. \\
+        \ac{NTP} & High-precision clocks, \newline mature protocol. & Requires internet. \\
         \ac{PTP} & Do not require internet, \newline accuracy in nanoseconds & Accuracy in nanoseconds only with cabled network \newline not very widespread among Android apps. \\ \bottomrule
     \end{tabularx}
     \caption{Pros and cons of the different synchronization methods.}\label{tab:pros_cons_sync}
 \end{table}
+
+
+
+\section{Refining the Synchronization Method --- SNTP}
+With the choice of synchronization method landing upon \ac{NTP}, we take a closer look at it, specifically in regard to \ac{SNTP}.
+As the name suggests \ac{SNTP} is based upon \ac{NTP} but vastly simplified, \ac{SNTP} trades accuracy and drift alleviation for simplicity and reduced resource requirements.
+\ac{SNTP} was developed as an \ac{NTP} alternative for small computers and micro-controllers, which happens to fit the description of smartphones.
+Furthermore we would not be taking full advantage of the utilities \ac{NTP} provides over \ac{SNTP} resulting i needless complexity.
+In reality, \ac{SNTP} is actually a better fit than \ac{NTP} considering how we want to use it, in order to understand why, first let us consider some of the major advantages which makes \ac{NTP} more accurate than \ac{SNTP} and why they are of little benefit to our case.\cite{sntp_ntp}
+\begin{description}
+    \item [Clock Skew] \hfill \\
+    Part of the algorithms \ac{NTP} incorporates are designed to alleviate clock skew.
+    While effective in real-time systems, where the device clock needs to be synchronized, it has no impact on our application due to us not sending real-time commands, but rather timestamps for when an event should occur.
+    \item [Drift Alleviation] \hfill \\
+    \ac{NTP} also alleviates by drift, it does this by speeding up or slowing down the clock, effectively introducing drift in the opposing direction such that it evens out.
+    As mentioned before, in order to adjust the clock through the application the device must be rooted, since we do not want to impose this requirement onto users, drift alleviation would be nullified as we cannot slow down or speed up the clock.

```

Comment:
Det er jo lige netop hele pointen, det er en begrundelse for hvorfor vi vil bruge SNTP istedet, eftersom nogle af de fordele som NTP har, ikke kan benyttes